packaging/linux/common/khiops

```diff
@@ -27,33 +24,38 @@ warning() {
     fi
 }
 
-if [[ ! -f /usr/bin/khiops-env ]]; then
-    error "/usr/bin/khiops-env is missing"
+get_script_dir(){
+    SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+    echo "$SCRIPT_DIR/"
+}
+
+if [[ ! -f "$(get_script_dir)"/khiops_env ]]; then
+    echo "$(get_script_dir)/khiops_env is missing"
+    exit 1
```

Comment:
I would consider moving this procedure to `khiops_env`, thus I think we could make do without these `khiops` scripts in the `khiops-core` Conda package. And this is a desirable thing IMHO, as there are several parts one does not need in the `khiops-core` Conda package (GUI, notify, etc).